bundle/src/assembly/resources/etc/runtime.json

```diff
@@ -213,7 +231,7 @@
                 "VehicleSlowDown",
                 "TrafficDetectorUpdates",
                 "TrafficLightUpdates",
-                "VehicleElectricityUpdates",
+                "VehicleBatteryUpdates",
```

Comment:
Shall we add VehicleChargingDenial, VehicleChargingStartRequest, and VehicleChargingStopRequest here too?